src/test/java/org/opensearch/security/securityconf/impl/v7/IndexPatternTests.java

```diff
@@ -108,47 +108,47 @@ public void testAttemptResolveIndexNamesOverload() {
     public void testExactNameWithNoMatches() {
         doReturn("index-17").when(ip).getUnresolvedIndexPattern(user);
         when(clusterService.state()).thenReturn(mock(ClusterState.class));
-        when(resolver.concreteIndexNames(any(), eq(IndicesOptions.lenientExpandOpen()), eq("index-17"))).thenReturn(new String[]{});
+        when(resolver.concreteIndexNames(any(), eq(IndicesOptions.lenientExpandOpen()), eq(true), eq("index-17"))).thenReturn(new String[]{});
```

Comment:
`eq(true)` - Why is this not needed to be validated before?

Reply:
The function call changed here: https://github.com/opensearch-project/security/commit/f7cc569c9d3fa5d5432c76c854eed280d45ce6f4#diff-77e2d2460234b7e8faa37a83d4a2eb672d0f5a4c678a229dc3a9c7e11c8d8b4aL785